### DISCUSSION TITLE
Allow escaping of dollar signs $

### DISCUSSION THREAD

**Discussion by RonSijm** [community/community#120728] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Lets say I have an example $text with dollar signs$ - I'm not able to escape that.

For example : \$text with dollar signs$\   - \\dollarsign

You are allowed to escape other styling formats, like italic : \*test\*  or \\backslashes\\ or \`backticks\` - but for some reason you can't escape dollar signs.

This behavior is the same across Github Issues, Github Discussions, and Github published .md files

**Comment by CopperEagle** [comment]
Use `\textdollar`. That way you can $get\space  the \space \textdollar \space \textdollar \space \textdollar$.  Yes, also $textwith\textdollar\textdollar\textdollar$.
